pegasus/sites.v3/code.org/public/markdown/advanced.md.erb

```diff
@@ -21,6 +21,48 @@ A table is created using vertical bars to separate columns. Use the vertical bar
 | [ScratchEd](http://scratched.gse.harvard.edu/guide) | A 6-unit intro to Scratch, FREE | In-person educator meet-ups and online MOOC, FREE |
 | [Tynker](https://www.tynker.com/school/lesson-plan) | Free tools, tutorials, and a 6-hr introductory lesson plan. 200+ lessons with assessments: $399/class, $2,000/school | 2-day PD, $2000/day + travel |
 
+## Partials
+***
+
+If you create your markdown template with the extension `.partial` in addition to the `.md` extension, you can use the partials syntax to embed other templates. The partials syntax simply requires the name of the template to embed wrapped in curly braces like so:
```

Comment:
Should it be called out that order matters for this? (I think order matters...)

Reply:
Order does sort of matter in that the order of the extension determines the order in which the processing logic happens, but for this instance both orders work fine.

The larger documentation update I'm working on will have a section talking specifically about combining extensions, and I'll have something there that talks about the effect of extension order.